### ISSUE TITLE
META-INF/versions/9/module-info flagged as duplicate class

### ISSUE THREAD

**Issue by renatoathaydes** [nebula-plugins/gradle-lint-plugin#246]
## gradle-lint plugin version

```groovy
id 'nebula.lint' version '12.0.1'
```

## Problem

I've just run into this issue after upgrading a dependency.

```
io.github.classgraph:classgraph:4.8.45 in configuration ':identityserver:host:compile' has 1 classes duplicated by org.apache.logging.log4j:log4j-api:2.12.0. Duplicate classes: [META-INF/versions/9/module-info]
```

which results in an error:

```
error     transitive-duplicate-dependency-classio.github.classgraph:classgraph:4.8.45 in configuration ':identityserver:host:compile' has 1 classes duplicated by org.apache.logging.log4j:log4j-api:2.12.0 (use --info for detailed class list) (no auto-fix available)
```

How can I ignore this for now?

**Comment by renatoathaydes** [comment]
> If you'd like to ignore that one violation, here's some info you can use so that you can wrap your dependency: https://github.com/nebula-plugins/gradle-lint-plugin/wiki/Ignoring-Violations

I tried something like this:

```groovy
dependencies {
    gradleLint.ignore {
        compile 'the conflicting dep'
    }
}
```

And that didn't work. Is that what you had in mind? Ignoring the rule entirely is not something I am willing to do.

**Comment by renatoathaydes** [comment]
> This is fixed in v12.1.0

Ah, nevermind then!! Will try. Thanks a lot.

**Comment by renatoathaydes** [comment]
It worked! Thanks again @OdysseusLives .
